presto-hive/src/test/java/com/facebook/presto/hive/metastore/TestingHiveMetastore.java

```diff
@@ -409,6 +413,30 @@ public synchronized void alterPartition(String databaseName, String tableName, P
     }
 
     @Override
+    public synchronized void createRole(String role, String grantor)
```

Comment:
I addressed that comment for FileBasedMetastore, but i think it doesn't matter here, because it is inmemory.